int64 action
regraspComm/Pick pick
regraspComm/Place place
regraspComm/Move move
regraspComm/ThrowToPalm throwToPalm
regraspComm/PushInEnveloping pushInEnveloping
regraspComm/Vibrate vibrate
regraspComm/ThrowAndFlip throwAndFlip
regraspComm/PushInFingers pushInFingers
regraspComm/RollToFingertip rollToFingertip
regraspComm/RollOnGround rollOnGround
regraspComm/RollToPalm rollToPalm
regraspComm/RollToGround rollToGround
regraspComm/Droop droopInFingers
regraspComm/ThrowToFingertip throwToFingertip
regraspComm/Squeeze squeeze
regraspComm/LieToStand lieToStand
regraspComm/StandToLie standToLie
regraspComm/LongEdgeDroop longEdgeDroop
regraspComm/ShortEdgeDroop shortEdgeDroop
regraspComm/Topple topple
regraspComm/Droop droop